lib/validation/validators/maxLength.ts

```diff
@@ -0,0 +1,5 @@
+import { helpers } from '@vuelidate/validators'
+
+export function maxLength(value: string, length: number): boolean {
+  return !helpers.req(value) || value.length <= length
```

Comment:
`!helpers.req(value)` may not be needed according to `createRule` implementation, but not so sure.

Reply:
Ah, OK. `validators` should not have this. Validator should always check the values.

And by the way, I think this can be on another PR but now that I think of, maybe all validator `value` arg should be `unknown` 🤔 Since it's validator, we should be able to pass anything.

```ts
maxLength(true) // <- false
```

Reply:
I created a follow-up issue (#438) and will handle in another PR.